src/main/java/com/karankumar/booksapi/repository/BookRepository.java

```diff
@@ -19,11 +19,15 @@
 import java.util.List;
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
 
 public interface BookRepository extends CrudRepository<Book, Long> {
   
     @Query("SELECT DISTINCT b FROM Book b LEFT JOIN FETCH b.authors")
     List<Book> findAllBooks();
 
+    Book findByBooktitle( String booktitle);
```

Comment:
This issue was meant to create a derived query to find books by genre, not title. Could you please change this?